src/components/Tabs/index.js

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react'
+import {Tab, Tabs, TabList, TabPanel} from 'react-tabs'
```

Comment:
I don't see `react-tabs` added in the package.json and yarn lock. You'll need to `yarn add react-tabs`.